Не запускается контейнер с админ-панелью

```
2024-11-21 22:24:02 2024-11-21T19:24:02.115Z  INFO 1 --- [           main] com.statistigz.cp.CpApplication          : Starting CpApplication v0.0.1-SNAPSHOT using Java 18.0.2 with PID 1 (/app/application.jar started by root in /app)
2024-11-21 22:24:02 2024-11-21T19:24:02.117Z  INFO 1 --- [           main] com.statistigz.cp.CpApplication          : No active profile set, falling back to 1 default profile: "default"
2024-11-21 22:24:02 2024-11-21T19:24:02.833Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2024-11-21 22:24:02 2024-11-21T19:24:02.977Z  INFO 1 --- [           main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 135 ms. Found 2 JPA repository interfaces.
2024-11-21 22:24:03 2024-11-21T19:24:03.434Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 9090 (http)
2024-11-21 22:24:03 2024-11-21T19:24:03.445Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-11-21 22:24:03 2024-11-21T19:24:03.446Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.20]
2024-11-21 22:24:03 2024-11-21T19:24:03.494Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-11-21 22:24:03 2024-11-21T19:24:03.495Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1309 ms
2024-11-21 22:24:03 2024-11-21T19:24:03.670Z  INFO 1 --- [           main] org.atmosphere.cpr.AtmosphereFramework   : Installed AtmosphereHandler com.vaadin.hilla.push.PushEndpoint mapped to context-path: /HILLA/push
2024-11-21 22:24:03 2024-11-21T19:24:03.671Z  INFO 1 --- [           main] org.atmosphere.cpr.AtmosphereFramework   : Installed the following AtmosphereInterceptor mapped to AtmosphereHandler com.vaadin.hilla.push.PushEndpoint
2024-11-21 22:24:03 2024-11-21T19:24:03.672Z  INFO 1 --- [           main] org.atmosphere.cpr.AtmosphereFramework   :   org.atmosphere.interceptor.AtmosphereResourceLifecycleInterceptor : Atmosphere LifeCycle
2024-11-21 22:24:03 2024-11-21T19:24:03.672Z  INFO 1 --- [           main] org.atmosphere.cpr.AtmosphereFramework   :   org.atmosphere.client.TrackMessageSizeInterceptor :  Track Message Size Interceptor using |
2024-11-21 22:24:03 2024-11-21T19:24:03.672Z  INFO 1 --- [           main] org.atmosphere.cpr.AtmosphereFramework   :   org.atmosphere.interceptor.SuspendTrackerInterceptor : UUID Tracking Interceptor
2024-11-21 22:24:03 2024-11-21T19:24:03.704Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing AtmosphereFramework
2024-11-21 22:24:05 2024-11-21T19:24:05.446Z  INFO 1 --- [           main] c.v.f.s.VaadinServletContextInitializer  : Search for subclasses and classes with annotations took 1585 ms
2024-11-21 22:24:05 2024-11-21T19:24:05.450Z ERROR 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Exception sending context initialized event to listener instance of class [com.vaadin.flow.spring.VaadinServletContextInitializer$CompositeServletContextListener]
2024-11-21 22:24:05 
2024-11-21 22:24:05 java.lang.RuntimeException: Unable to initialize com.vaadin.flow.spring.VaadinServletContextInitializer$DevModeServletContextListener
2024-11-21 22:24:05     at com.vaadin.flow.spring.VaadinServletContextInitializer$FailFastServletContextListener.contextInitialized(VaadinServletContextInitializer.java:206) ~[vaadin-spring-24.5.4.jar!/:na]
2024-11-21 22:24:05     at com.vaadin.flow.spring.VaadinServletContextInitializer$CompositeServletContextListener.lambda$contextInitialized$0(VaadinServletContextInitializer.java:230) ~[vaadin-spring-24.5.4.jar!/:na]
2024-11-21 22:24:05     at java.base/java.util.ArrayList.forEach(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at com.vaadin.flow.spring.VaadinServletContextInitializer$CompositeServletContextListener.contextInitialized(VaadinServletContextInitializer.java:230) ~[vaadin-spring-24.5.4.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4450) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4890) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1332) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1322) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at java.base/java.util.concurrent.AbstractExecutorService.submit(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:866) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:845) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1332) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1322) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at java.base/java.util.concurrent.FutureTask.run(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at java.base/java.util.concurrent.AbstractExecutorService.submit(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:866) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:240) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardService.startInternal(StandardService.java:433) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:921) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:171) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.apache.catalina.startup.Tomcat.start(Tomcat.java:437) ~[tomcat-embed-core-10.1.20.jar!/:na]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:126) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:105) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:499) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:218) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:188) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:162) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:618) ~[spring-context-6.1.6.jar!/:6.1.6]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:334) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1354) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1343) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at com.statistigz.cp.CpApplication.main(CpApplication.java:10) ~[!/:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:91) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:53) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.JarLauncher.main(JarLauncher.java:58) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05 Caused by: java.lang.IllegalStateException: Failed to determine project directory for dev mode. Directory '/app' does not look like a Maven or Gradle project. Ensure that you have run the prepare-frontend Maven goal, which generates 'flow-build-info.json', prior to deploying your application
2024-11-21 22:24:05     at com.vaadin.flow.server.AbstractConfiguration.getProjectFolder(AbstractConfiguration.java:238) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.flow.server.AbstractConfiguration.getJavaResourceFolder(AbstractConfiguration.java:261) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.experimental.FeatureFlags.getFeatureFlagFile(FeatureFlags.java:368) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.experimental.FeatureFlags.loadProperties(FeatureFlags.java:207) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.experimental.FeatureFlags.lambda$get$0(FeatureFlags.java:169) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.flow.server.VaadinServletContext.getAttribute(VaadinServletContext.java:73) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.experimental.FeatureFlags.get(FeatureFlags.java:164) ~[flow-server-24.5.4.jar!/:24.5.4]
2024-11-21 22:24:05     at com.vaadin.base.devserver.startup.DevModeInitializer.initDevModeHandler(DevModeInitializer.java:193) ~[vaadin-dev-server-24.5.4.jar!/:na]
2024-11-21 22:24:05     at com.vaadin.base.devserver.DevModeHandlerManagerImpl.initDevModeHandler(DevModeHandlerManagerImpl.java:100) ~[vaadin-dev-server-24.5.4.jar!/:na]
2024-11-21 22:24:05     at com.vaadin.flow.spring.VaadinServletContextInitializer$DevModeServletContextListener.failFastContextInitialized(VaadinServletContextInitializer.java:591) ~[vaadin-spring-24.5.4.jar!/:na]
2024-11-21 22:24:05     at com.vaadin.flow.spring.VaadinServletContextInitializer$FailFastServletContextListener.contextInitialized(VaadinServletContextInitializer.java:202) ~[vaadin-spring-24.5.4.jar!/:na]
2024-11-21 22:24:05     ... 46 common frames omitted
2024-11-21 22:24:05 
2024-11-21 22:24:05 2024-11-21T19:24:05.452Z ERROR 1 --- [           main] o.apache.catalina.core.StandardContext   : One or more listeners failed to start. Full details will be found in the appropriate container log file
2024-11-21 22:24:05 2024-11-21T19:24:05.453Z ERROR 1 --- [           main] o.apache.catalina.core.StandardContext   : Context [] startup failed due to previous errors
2024-11-21 22:24:05 2024-11-21T19:24:05.471Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2024-11-21 22:24:05 2024-11-21T19:24:05.472Z  WARN 1 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Unable to start web server
2024-11-21 22:24:05 2024-11-21T19:24:05.480Z  INFO 1 --- [           main] .s.b.a.l.ConditionEvaluationReportLogger : 
2024-11-21 22:24:05 
2024-11-21 22:24:05 Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2024-11-21 22:24:05 2024-11-21T19:24:05.492Z ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed
2024-11-21 22:24:05 
2024-11-21 22:24:05 org.springframework.context.ApplicationContextException: Unable to start web server
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:165) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:618) ~[spring-context-6.1.6.jar!/:6.1.6]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:334) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1354) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.SpringApplication.run(SpringApplication.java:1343) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at com.statistigz.cp.CpApplication.main(CpApplication.java:10) ~[!/:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[na:na]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:91) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:53) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05     at org.springframework.boot.loader.launch.JarLauncher.main(JarLauncher.java:58) ~[application.jar:0.0.1-SNAPSHOT]
2024-11-21 22:24:05 Caused by: org.springframework.boot.web.server.WebServerException: Unable to start embedded Tomcat
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:145) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.<init>(TomcatWebServer.java:105) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getTomcatWebServer(TomcatServletWebServerFactory.java:499) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory.getWebServer(TomcatServletWebServerFactory.java:218) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.createWebServer(ServletWebServerApplicationContext.java:188) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.onRefresh(ServletWebServerApplicationContext.java:162) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     ... 13 common frames omitted
2024-11-21 22:24:05 Caused by: java.lang.IllegalStateException: StandardEngine[Tomcat].StandardHost[localhost].TomcatEmbeddedContext[] failed to start
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.rethrowDeferredStartupExceptions(TomcatWebServer.java:207) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.initialize(TomcatWebServer.java:129) ~[spring-boot-3.2.5.jar!/:3.2.5]
2024-11-21 22:24:05     ... 18 common frames omitted
2024-11-21 22:24:05
```

--- a/.github/workflows/build-push.yml
+++ b/.github/workflows/build-push.yml
@@ -22,7 +22,7 @@ jobs:
           cache: maven
           
       - name: Build with Maven
-        run: mvn -B clean package -DskipTests
+        run: mvn -B clean package -DskipTests -Pproduction
         working-directory: ${{env.working-directory}}
         
       - name: Log in to Docker Hub

